201 new figure figure 14

aassuied-ps opened the pull request:
Closes #201

edelarua reviewed (approved):
Thanks @aassuied-ps! I added a parameter for the visit variable and fixed the ordering of the arm levels in the table, so it should be good to go now!! :)

aassuied-ps commented:
> Thanks @aassuied-ps! I added a parameter for the visit variable and fixed the ordering of the arm levels in the table, so it should be good to go now!! :)

Thanks a lot @edelarua !
